.github/workflows/r_artifacts.yml

```diff
@@ -37,3 +49,9 @@ jobs:
         with:
           name: ${{ steps.build_package.outputs.artifact_name }}
           path: ${{ steps.build_package.outputs.artifact_path }}
+      - name: Send final status
+        if: ${{ always() }}
+        run: |
+          $GITHUB_WORKSPACE/.ci/append_comment.sh \
+            "${{ github.event.client_payload.comment_number }}" \
+            "Status: ${{ job.status }}."
```

Comment:
Yes, I know that direct link for artifact downloading will be more useful. But unfortunately, it is impossible: 
> Gets a redirect URL to download an archive for a repository. This URL expires after 1 minute.
   https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#download-an-artifact

Reply:
ha it's ok, we don't use this job that often and it's only one or two more clicks to get to the artifact